packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -722,6 +724,15 @@ export default decorate([
                       </Tooltip>
                     )}
                   </div>
+                  <a
+                    className='text-muted'
+                    href={BACKUP_NG_DOC_LINK}
+                    rel='noopener noreferrer'
+                    target='_blank'
+                  >
+                    <Icon icon='info' />{' '}
+                    {_('backupNgLinkToDocumentationMessage')}
+                  </a>
```

Comment:
Add a `<p className='text-muted' />` around the link.

And I'm wondering if it would not be better centered (the icon pokes out of the buttons).

Reply:
`className='text-muted'` in the `p` tag will not change the default style of the link.

Reply:
Ok, add a paragraph without the class then :)

Reply:
The icon pokes out of the buttons because the button row has a left margin/padding (that should probably not be there IMO). We shouldn't fix the buttons issue by centering the tip :)

Reply:
Or are the buttons also centered? :thinking:

Reply:
They are and I think it's best.